src/new-components/heroes/heroes.stories.mdx

```diff
@@ -57,3 +57,44 @@ optionally with headline and text.
 This works great
 </ImageHero>
 ```
+
+# Blog Hero
+> Due to the gatsby image dependency no preview is available.
+
+This hero behaves very similar to the `ImageHero` but you can pass in two additional things: `attribution` and a squared image
+that will be used once the website is very narrow (higher than wide) to present a more pleasant part from the feature image.
+
+
+```
+<BlogHero
+  attribution={
+    {
+      creator: "My Source;
+      source: "http://www.example.com;
+    }
+  }
+  image={featuredImage}
+  imageSquare={featuredImageSquared}
+/>
+```
+
+
+You can query for both images in a single graphql query. Pass `aspectRatio: 1` to get a square image.
+
+```
```

Comment:
```suggestion
```graphql
```

Reply:
I did not expect to get colored code here, that's much better 😅 Thanks